src/legacy/core_plugins/vis_type_timeseries/server/lib/vis_data/__tests__/helpers/get_es_shard_timeout.js

```diff
@@ -25,7 +25,7 @@ describe('getEsShardTimeout', () => {
     const req = {
       getEsShardTimeout: async () => {
         return 12345;
-      }
+      },
```

Comment:
ugh lol

Reply:
I know right...dunno why it didn't tell me in the first place 🤔